client/components/jetpack/scan-threats/index.tsx

```diff
@@ -174,9 +174,8 @@ const ScanThreats = ( { error, site, threats }: Props ) => {
 						count: threats.length,
 					}
 				) }
-				<br />
```

Comment:
If we remove this `br` we should add a space at the end of the lines above. This is how it looks at the moment:

![image](https://user-images.githubusercontent.com/390760/98689806-305c7700-2364-11eb-9874-c9300653bea2.png)

Reply:
Thanks, I have my test version working now too, updated.